docs/devices.rst

```diff
@@ -60,6 +60,42 @@ type of device. The available iterators are :py:class:`DeviceTokenList` and :py:
    :members: DeviceTokenList, APIDList, DeviceInfo
    :noindex:
 
+
+Open Channel Registration
+--------------
+
+Open Channels are registered by using :py:class:`OpenChannel`.
+
+.. code-block:: python
+
+   import urbanairship as ua
+   airship = ua.Airship(app_key, app_secret)
+
+   my_channel = ua.OpenChannel()
+   my_channel.address = 'my_email@example.com'
+   my_channel.open_platform = 'email'
+   my_channel.opt_in = True
+   my_channel.create(airship)
```

Comment:
Then again, it does actually register successfully and shows up as a 200 in our logs. Not sure where the 404 is coming from. This is what I'm doing and getting (after setting up airship, etc.): 
```In [28]: new_channel = ua.OpenChannel()

In [29]: new_channel.address = 'test+a+newchannel@example.com'

In [30]: new_channel.open_platform = 'email'

In [31]: new_channel.opt_in = True

In [33]: new_channel.create(airship)
---------------------------------------------------------------------------
AirshipFailure                            Traceback (most recent call last)
<ipython-input-33-23ce52e37289> in <module>()
----> 1 new_channel.create(airship)

.../dev/python-library/urbanairship/devices/open_channel.py in create(self, airship, set_tags, hydrate)
     70
     71         if hydrate:
---> 72             payload = self._get_payload_for_id(airship, self.channel_id)
     73             self.from_payload(payload)
     74

.../dev/python-library/urbanairship/devices/open_channel.py in _get_payload_for_id(airship, channel_id)
    153             body=None,
    154             url=url,
--> 155             version=3
    156         )
    157         return response.json()['channel']

.../dev/python-library/urbanairship/core.pyc in _request(self, method, body, url, content_type, version, params, encoding)
     63             raise common.Unauthorized
     64         elif not (200 <= response.status_code < 300):
---> 65             raise common.AirshipFailure.from_response(response)
     66
     67         return response

AirshipFailure: (404, '{"ok":false,"error":"Entity not found","error_code":40401}')```